src/components/TablePackages/PackageTableRow.tsx

```diff
@@ -0,0 +1,19 @@
+import { CheckBoxOutlineBlankOutlined } from "@mui/icons-material";
+import { TableCell, TableRow } from "@mui/material";
+import { Packages } from "./types";
+
+const PackageTableRow = ({ id, kindness, ...rest }: Packages) => {
+  return (
+    <TableRow>
+      <TableCell>
+        <CheckBoxOutlineBlankOutlined />
+      </TableCell>
+      {Object.entries(rest).map(([key, element]) => {
+        return <TableCell key={key}>{element}</TableCell>;
+      })}
```

Comment:
Please don't display all fields here - it will be much harder for others to add their changes. It will be enough if you display just one TableCell with Checkbox